Lands of the Bohemian Crown/jsons/translations/French.properties

```diff
@@ -0,0 +1,52 @@
+Lands of the Bohemian Crown = Couronne de Bohême
+Charles IV. = Charles IV
+Bohemian = bohémien
+O King of all Czechs, Moravian and Silesians. Thou who are the greatest of us Czechs who have ever lived. Holy Roman Emperor. Your whole life is synonym of peace, prosperity and diligence. It was under your hand that our country began to flourish. You built a stone bridge over the Vltava river, who founded a big european university and you, who fed the hungry when building the Hungry wall. There will never be a greater ruler than you. = Ô Roi des Tchèques, Moraves et Silésiens. Toi le plus grand Tchèque qui ai jamais vécu. Empereur du Saint-Empire Romain Germanique. Ta vie fut synonyme de paix, prospérité et diligence. Sous ton règne ton pays a fleuri. Tu as bâti un pont de pierre sur la Vltava, fondé une grande université et nourri les affamés. Il n'y aura jamais plus grand dirigeant que toi.
+The most beloved of us, benevolent Charles, once again lead our nation to prosperity. Bring our nation to the Roman throne and do not leave us. Raise your bow once more while hunting in your rich forests. After your death, there will no longer be as great ruler as you are. You will remain unforgettable. = Charles bien-aimé et bienveillant, mène ta nation à la prospérité une dernière fois. Amène-la au trône de Rome et ne nous quitte plus. Lève encore ton arc pour chasser dans tes fiches forêts. Après ta mort il n'y aura pas de plus grand dirigeant que toi. Tu resteras inoubliable.
+In the Hearth of Europe = Au coeur de l'Europe
+
+Prague = 
+Wrocław = 
+Brünn = 
+Ostrawa =
+Katowicy = 
+Pilsen = 
+Uopole = 
+Reichenberg = 
+Chóśebuz = 
+Ołomuniec = 
+Budweis = 
+Aussig an der Elbe =
+Königgrätz = 
+Pardubitz = 
+Hawierzów = 
+Kladen = 
+Brüx = 
+Friedeck-Mistek = 
+Racibōrz = arwina =
```

Comment:
Mistake?

Reply:
Oh I don't where that comes from
Yeah, mistake